engine/access/rest/select_filter_test.go

```diff
@@ -0,0 +1,189 @@
+package rest
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/onflow/flow-go/engine/access/rest/generated"
+)
+
+func ExampleSelectFilter() {
```

Comment:
this example is great, but we should also have explicit unit tests.

Reply:
Added `TestSelectFilter` as an explicit test (in addition to this Example test)